### PR TITLE
School Closure support

### PR DESCRIPTION
**Pull request by maneshkommidi**
Provide `school_closing` to the autocomplete endpoint, then `school_closing`, `school_closing_date`, and `school_closing_message` to the institution profile endpoint.

Local Reference: https://github.com/department-of-veterans-affairs/gibct-data-service/issues/322
Consumer: Not yet provided
Internal: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9369

### CODE REVIEW

File: db/migrate/20180510145019_create_school_closures.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CreateSchoolClosures < ActiveRecord::Migration
+  def change
+    create_table :school_closures do |t|
+      t.string :facility_code, null: false
+      t.string :institution_name
+      t.boolean :school_closing
```
[the remark this answers is not in the document]
Comment (maneshkommidi) [reply db/migrate/20180510145019_create_school_closures.rb]: Sure @jkassemi I think on institutions model we will add new column school_closing with default value false, the value of  school_closing will be true only if facility_code matches to any of the facility_code in school_closures model.

[the remark this answers is not in the document]
Comment (jkassemi) [reply db/migrate/20180510145019_create_school_closures.rb]: Is there any case where EDU will provide a school closures CSV that will contain a school_closing value of `false`? Can you follow up with the team for clarification?

[the remark this answers is not in the document]
Comment (maneshkommidi) [reply db/migrate/20180510145019_create_school_closures.rb]: I will check @briangryth regarding this.

[the remark this answers is not in the document]
Comment (briangryth) [reply db/migrate/20180510145019_create_school_closures.rb]: email confirmation received on 5/23. We expect that the school_closing value will always be true. However, we should account for the possibility that the value could be "false"

[the remark this answers is not in the document]
Comment (jkassemi) [reply db/migrate/20180510145019_create_school_closures.rb]: Cool. @maneshkommidi, let's go `null: false` for this one. I'm fine on not specifying the default (the value should be set on import and an error should be presented when neither true nor false are used, so the `false` default will be fine).

**Review by saneshark — APPROVED** [review]
LGTM

**Pull request merged**